### ISSUE TITLE
Upload collection accessibility issues

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#4670]
Upload collection accessibility issues

Attached the doc

[Upload collection.docx](https://github.com/SAP/fundamental-ngx/files/5980098/Upload.collection.docx)

**Comment by Sanjay-1312** [comment]
Issue 1 Fixed
Issue 2: Fixed
Issue 3: Fixed
Issue 4:
OB: On activating the 'New Folder' button, the dialog appears and the focus should be on the 'Cancel' button and additional information such as 'New Folder Cancel button is read.
EB: Ideally, the focus should be available on the 'Input field' and only cancel button should be read when accesed.
![image](https://user-images.githubusercontent.com/78714518/127634827-7b4ef1c8-7347-4a68-864c-d7e1f8037b23.png)
Issue 5: Fixed
Issue 6:Fixed
Issue 7: Fixed
Issue 8: Fixed
Issue 9: Fixed

**Comment by Sanjay-1312** [comment]
Issue:
OB: On activating the 'New Folder' button, the details of the dialog read twice to the user.
EB: Ideally, the dialog details must be read once to the user.
![image](https://user-images.githubusercontent.com/78714518/128116530-0efa5902-fc89-4780-b4c2-d9ddf8128985.png)

Issue:
OB: On accessing the input field, additional information such as dialog name 'New Folder' sis read.
EB: Irrespective of the directions in which it is accessed, only relevant and visible information about the UI should be read out and such extra information should be removed.
![image](https://user-images.githubusercontent.com/78714518/128116776-7a530a46-bedc-4b2f-9a88-e40d8f544c1f.png)

**Comment by kavya-b** [comment]
Hi @Sanjay-1312,
Could you please verify these 2 issues again against this url:  https://deploy-preview-6118--fundamental-ngx.netlify.app/#/platform/upload-collection  ? Thank you!

**Comment by Sanjay-1312** [comment]
Issue 2 
Fixed.

**Comment by kavya-b** [comment]
issue 1 seems to more of Dialog component issue. Even now, https://fundamental-ngx.netlify.app/#/core/dialog reads the dialog heading twice for some examples, like [this with inner popover](https://fundamental-ngx.netlify.app/#/core/dialog#popover-inside-dialog), [custom backdrop button example](https://fundamental-ngx.netlify.app/#/core/dialog#custom-backdrop-container-configuration), [stacked dialog example](https://fundamental-ngx.netlify.app/#/core/dialog#stacked), [complex example](https://fundamental-ngx.netlify.app/#/core/dialog#complex), [mobile example](https://fundamental-ngx.netlify.app/#/core/dialog#mobile). I don't think this issue can be fixed with this PR, @Sanjay-1312 , could you please add this issue while testing Dialog accessibility instead?

**Comment by kavya-b** [comment]
https://github.com/SAP/fundamental-ngx/issues/6100 issues 2 and 5 correctly describe the problem happening here. @Sanjay-1312

**Comment by kavya-b** [comment]
blocked until #6100 issues are fixed
